Editor/Window/ManagedEC2Page.cs

```diff
@@ -106,8 +106,8 @@ private void UpdateGUI()
                 _deploymentSettings.CurrentStackInfo.StackStatus is StackStatus.CreateComplete
                     or StackStatus.UpdateComplete);
 
-            _deploymentScenariosInput.SetEnabled(_deploymentSettings.CanDeploy);
-            _fleetParamsInput.SetEnabled(_deploymentSettings.CanDeploy);
+            _deploymentScenariosInput.SetEnabled(_stateManager.IsBootstrapped);
+            _fleetParamsInput.SetEnabled(!_deploymentSettings.IsDeploymentRunning); 
```

Comment:
This seems like a potential bug. Looks like `CanDeploy` checks more conditions outside of just whether the profile is bootstrapped or not 

```
        public bool CanDeploy => !IsDeploymentRunning && IsBootstrapped && IsFormFilled && IsCurrentStackModifiable;
```

Reply:
I have introduced a new Property `CanEdit` to handle when these fields should be editable